js/data-desk-search.js

```diff
@@ -198,8 +195,22 @@ $(function() {
           $output.find("#" + dataset.id).replaceWith(datasetTemplate(dataset));
         });
     });
+  };
 
+  $("#corporate-data-search form").on('submit', function(e) {
+    e.preventDefault();
+    var q = queryInput.val();
+    history.pushState(q, "TRACE search: " + q, "trace.html?q=" + q);
```

Comment:
good point. chrome encoded it for me but it does seem like it could be an issue so encoding explicitly to be sure.